### PR TITLE
Add support for markdown math formulas

### PR DESCRIPTION
**Pull request by kristofferlund**
##### Checklist

- [x] PR description included and it is linked to an issue #58 
- [x] `make test` and `make lint` passes
- [x] relevant documentation is changed or added

Notes:

### CODE REVIEW

File: core/examples/minimal_example.py [hunk 1]
```diff
@@ -19,16 +19,30 @@
...
+
+You can read more about the input format
+[here](http://docs.mathjax.org/en/latest/tex.html#).
 
+Example:
+
+`formula = r'$$x_{1,2} = \frac{-b \pm \sqrt{b^2-4ac}}{2b}.$$'`
+
+Renders out to this:
 """)
```
Comment (anders-kiaer) [inline core/examples/minimal_example.py]: Suggestion for example showing different ways of entering equations, labelling and corresponding equation reference, in-line symbol/equations and multi-line equations:

```
If you want to use math formulas, you have several different options. You can
use double dollar signs,

$$ \left(\frac{\sqrt x}{y^3}\right), $$

or you can wrap it inbetween special commands like this:

\begin{equation}
\cos (2\theta) = \cos^2 \theta - \sin^2 \theta \label{my_cos_equation}.
\end{equation}

All equations with labels can easily be referred to in the text as
\eqref{my_cos_equation}. If you want an equation without numbering add "notag":

\begin{equation}
\lim_{x \to \infty} \exp(-x) = 0.\notag
\end{equation}

If you want to write multi-line equations aligned on e.g. the equal sign, you 
can also do that:

\begin{align}
f(x) &= (x+a)(x+b) \\\\
     &= x^2 + (a+b)x + ab
\end{align}

The & operator indicates what to align on. You can also write in-line equations
or symbols inbetween, like \\( \alpha \\) and \\( \gamma \\).

To prevent build failing because of backslashes, use a rawstring format by adding `r` in front of the string.

You can read more about the input format [here](http://docs.mathjax.org/en/latest/tex.html#).

Example:

`formula = r'$$x_{1,2} = \frac{-b \pm \sqrt{b^2-4ac}}{2b}.$$'`

Renders out to this:
```
I had to put two backslashes when they were followed by \, ( or ), compared with MathJax documentation, to get it to work. Might have something to do with the markdown processing before it is given to MathJax.

File: core/webviz/_math.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                         "a11y/accessibility-menu.js"
+                    ],
+                    TeX: {
+                        extensions: [
+                            "AMSmath.js",
+                            "AMSsymbols.js",
+                            "noErrors.js",
+                            "noUndefined.js"
+                        ]
+                    }
```
Comment (anders-kiaer) [inline core/webviz/_math.py]: If the `TeX` dictionary is changed to this we get numbering of equations (which then can be referenced):

```
                    TeX: {
                        equationNumbers: {autoNumber: "all"},
                        extensions: [
                            "AMSmath.js",
                            "AMSsymbols.js",
                            "noErrors.js",
                            "noUndefined.js"
                        ]
                    }
```

File: core/webviz/_markdown.py [hunk 3]
```diff
@@ -15,14 +16,61 @@ def __init__(self, md):
         """
         super(Markdown, self).__init__()
         self._md = md
+        math_detected = False
 
-        self._rendered = markdown.markdown(
-            self._md,
+        if '$$' or r'\begin' or r'\end' in self._md:
+            math_detected = True
```
Comment (eivindjahren) [inline core/webviz/_markdown.py]: Probably good idea to write this as:

```
def contains_math(str):
    return '$$' or r'\begin' or r'\end' in str
...
if contains_math(self._md):
    self.header_elements.add(...
```

Comment (eivindjahren) [reply core/webviz/_markdown.py]: You could also look at the rendered output:

```
   def contains_math(str):
      return '<p class=...' in str
```
 where the class is whatever the mdx_math inserts.

File: core/examples/minimal_example.py [hunk 4]
```diff
@@ -19,15 +19,80 @@
...
+Result: $$ \left(\frac{\sqrt x}{y^3}\right) $$
+
+Or you can wrap it between special commands like this:
+
+```
+\begin{equation}
+\cos (2\theta) = \cos^2 \theta - \sin^2 \theta \label{my_cos_equation}.
+\end{equation}
+```
+
```
Comment (anders-kiaer) [inline core/examples/minimal_example.py]: Good refactoring of the example :+1: Maybe also add `Result:` here?

File: core/examples/minimal_example.py [hunk 5]
```diff
@@ -19,15 +19,80 @@
...
+All equations with labels can easily be referred to in the text as
+```\eqref{my_cos_equation}```. If you want an equation without numbering
+add "notag":
```
Comment (anders-kiaer) [inline core/examples/minimal_example.py]: ...and then this paragraph can be changed to:

All equations with labels can easily be referred to in the text as
```\eqref{my_cos_equation}```, resulting in \eqref{my_cos_equation}. 

If you want an equation without numbering add `\notag`:

File: core/webviz/_markdown.py [hunk 6]
```diff
@@ -35,6 +37,7 @@ def __init__(self, md):
         )
 
         if contains_math(self._md):
+            print('it contains math')
```
Comment (eivindjahren) [inline core/webviz/_markdown.py]: this print statement should probably not be here

File: core/webviz/_markdown.py [hunk 7]
```diff
@@ -7,7 +7,9 @@
 
 
 def contains_math(string):
-    if '$$' or r'\begin' or r'\end' in string:
+    print(string)
```
Comment (eivindjahren) [inline core/webviz/_markdown.py]: This print statement should probably not be here
